components/phpunit_bridge.rst

```diff
@@ -0,0 +1,129 @@
+.. index::
+   single: PHPUnitBridge
+   single: Components; PHPUnitBridge
+
+The PHPUnit Bridge Component
+=============================
+
+    The PHPUnit Bridge component provides utilities to report trigerred errors,
+    legacy tests and usage of deprecated code.
```

Comment:
Btw, the clock mock mocks the time functions (`time()`, `microtime()`, `sleep()` and `usleep()`). This way, you can test methods relying on time very accurately.

For instance. Assume you have some test like this:
```php
$stopwatch = new Stopwatch();

$stopwatch->start();
sleep(1);
$duration = $stopwatch->stop();

$this->assertEquals(1, $duration);
```
Without the clock mock, this can return in failure as `$duration` might be `1.000023` (for example). If you use the clock mock, this will always return exactly `1` (as `sleep()` will just do `startTime + 1` instead of really going to sleep).

To enable the clockmock, annotate your test with `@group time-sensitive` or call `ClockMock::register(true)` (before test) and `ClockMock::register(false)` (after test) yourself.

I think the feature is pretty great and worth documenting (it doesn't have to be included in this PR though).